src/stacks/typescript/index.ts

```diff
@@ -7,7 +7,10 @@ export class TypeScript {
   async isAvailable() {
     try {
       const packageJSON = require(`${this.rootPath}package.json`);
-      return Object.keys(packageJSON.devDependencies).includes('typescript');
+      if (packageJSON.devDependencies !== undefined) {
```

Comment:
We should maybe create an utility method in order to check if the project as something as a dep or devDep